frontend/src/AlertsManagement/AlertsManagement.css

```diff
@@ -1,3 +1,15 @@
-.alerts-management h1 {
+.alerts-management h1, h2, h3, h4 {
   text-align: center;
 }
+
+.alerts-management #alerts-header {
+  margin-top: 50px;
+}
+
+.alerts-management .alert-info {
+  width: 80%;
+  margin: 0 auto;
+}
+.alerts-management .alert-info #back-button {
```

Comment:
nit: add new line

Reply:
I put them together because the `#back-button` id is like a child of the previous section (`.alerts-management .alert-info`)